src/utils/jwtConfiguration.ts

```diff
@@ -0,0 +1,13 @@
+import jwt from 'jsonwebtoken'
+
+export const generateToken = (data: object) => {
+  const token = jwt.sign(data, process.env.JWT_SECRET || 'USER-AUTH', {
```

Comment:
```suggestion
  const token = jwt.sign(data, JWT_SECRET, {
```
its better to import enviroment variables from `src/config`